distributed/tests/test_client.py

```diff
@@ -4612,6 +4612,12 @@ def f(x):
         del distributed.tmp_client
 
 
+def test_get_client_no_cluster():
+    msg = u'No global client found and no address provided'
```

Comment:
Seems fine to me.  I recommend dropping the unicode string prefix here.  The dask codebase defaults to Python 3.

Reply:
Okay, fixed.